wordpress-mobile/WordPress-Android
# Implement React Native bridge method for making api requests

This PR implements a minimal api call to allow gutenberg to request from javascript the api call necessary to retrieve an image's available sizes.

Related PRs:
- gutenberg-mobile: wordpress-mobile/gutenberg-mobile#1455
- fluxc: https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1424


### To test:

Perform this test **with both (1) a WP.com site and (2) a self-hosted site**. It should work for both:

1. Run metro server on wordpress-mobile/gutenberg-mobile#1455 (`git submodule update --init --recursive`)
2. Add an image block to gutenberg.
3. Select an image from the WordPress media Library.
4. Select settings and choose a different image setting.
5. Observe the image adjust it's visual size if appropriate
6. Change to HTML mode.
7. Check that the image URL has changes according to the selected image size.

### TODO before merge

- [ ] update fluxc hash to tag (perhaps a beta release tag)
- [ ] update gutenberg-mobe hash to merged gutenberg-mobile hash

PR submission checklist:
